src/arkhamdb/ArkhamDb.ttslua

```diff
@@ -198,7 +198,7 @@ do
       local altArt     = { front = "normal", back = "normal" }
 
       -- translating front ID
-      if altFrontId > 90000 and altFrontId < 90050 then
```

Comment:
This is a pretty serious nightmare of maintenance. Are there other ways to determine parallel or not?

Reply:
Very valid concern and this was indeed a pain in the ass to implement, but unfortunately the ArkhamDB Api just provides a single altId. We have to use this to discern whether this means it's a parallel investigator, revised core alt art or promo art.